scripts/module/keybinder.nut

```diff
@@ -0,0 +1,54 @@
+keys <- {
+  "a" : 0x41,
+    "b" : 0x42,
+    "c" : 0x43,
+    "d" : 0x44,
+    "e" : 0x45,
+    "f" : 0x46,
+    "g" : 0x47,
+    "h" : 0x48,
+    "i" : 0x49,
+    "j" : 0x4A,
+    "k" : 0x4B,
+    "l" : 0x4C
+    }; // FIXME: Complete the table
+
+
+binds <- array(100, null);
+
+function onPlayerCommand( player, cmd, text ) {
+  cmd = cmd.tolower();
+  if (cmd == "b") {
+    text = text.tolower();
+    local bindTable = {};
+    local textArr = split(text, " ");
+    if (textArr.len() < 2) {
+      MessagePlayer("Usage: /" + cmd + " <key> <command>", player);
+    } else if (!(textArr[0] in keys)) {
```

Comment:
You should probably allow users to enter raw scancodes, for multimedia keys and whatever.